app/src/main/res/layout/item_settings_link.xml

```diff
@@ -0,0 +1,41 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <RelativeLayout
+        android:id="@+id/linkWrapper"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="?attr/selectableItemBackground"
+        android:clickable="true"
+        android:focusable="true"
+        android:padding="@dimen/default_screen_padding">
+
+        <ImageView
+            android:id="@+id/linkIcon"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentStart="true"
+            android:layout_centerInParent="true"
+            android:contentDescription="@string/content_description_back_button"
+            android:src="@drawable/ic_edit_profile" />
+
+        <TextView
+            android:id="@+id/linkText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:layout_marginStart="@dimen/big_margin"
+            android:layout_toStartOf="@+id/btnLink"
+            android:layout_toEndOf="@+id/linkIcon"
+            tools:text="Profil bearbeiten" />
+
+        <ImageView
+            android:id="@+id/btnLink"
```

Comment:
Button meines Erachtens nicht notwendig (Item heißt ja schon "XY bearbeiten")

Reply:
Würde den Button dennoch beibehalten, da er oftmals in anderen populären Apps vorzufinden ist und somit Wiedererkennungswert hat und Leute somit wissen, was passiert, wenn man darauf klickt.